### DISCUSSION TITLE
Latest "Modern" Style (Onion Skin) over Classic, paradox.

### DISCUSSION THREAD

**Discussion by adonios77** [Open-Shell/Open-Shell-Menu#2063] [Ideas]
The new modern style images in the 'Start Menu Style' field did not help. (I think it's called 'Onion Skin'). The old ones were fine, and accurately depicted the changes that were about to happen to the menu layout (ex: now scroll bars and tree style menu don't show etc.). Even more, this latest 'modern' style appearance over the 'Classic-style-menu' program is a paradox.
I think that one of the reasons this program was created (and loved a lot) was its **usefulness against 'modern' UIs** whose 'simplification' was either not self-explanatory enough (or even hiding previous features, but that's another topic).
_I'm talking about the "Start Menu Style" changes on Open-Shell Menu v4.4.194_ 
![](https://raw.githubusercontent.com/Open-Shell/Open-Shell-Menu/c517e21465cc475ae125c28b1f71030756f6a8a9/Src/StartMenu/StartMenuDLL/style_7.png) ![](https://raw.githubusercontent.com/Open-Shell/Open-Shell-Menu/f60e2a18cbc6e3ad3db4da74d2e58d528b0d1e4c/Src/StartMenu/StartMenuDLL/style_7.png)

**Comment by ge0rdi** [comment]
The purpose of these _style_ pictures is (and always was) to show differences between various styles.
And I think these new pictures do that very well (in nice schematic way).

They never depicted how actual start menu will look like, because that is heavily dependent on used skin.

> I think that one of the reasons this program was created (and loved a lot) was its usefulness against 'modern' UIs whose 'simplification' was either not self-explanatory enough (or even hiding previous features, but that's another topic).

That's one of reasons, but not the only one. Users still have great control over their start menu appearance, there was no change to that.

Also one can have modern looking start menu (even simplified if they want) with all the useful functionality that was removed from official Windows start menu.

**Reply by Darthagnon** [reply]
> They never depicted how actual start menu will look like

They did tho

If I want a Win10-styled start menu, I don't install OpenShell. I'm reinstalling it now, as I have tired of the Win10 start menu again. Modifying the DLL again to revert the inaccurate previews added in #1836

**Comment by CTVCAM8** [comment]
Looks fresh.
For some, It might be a little more difficult to adjust to a simpler way of viewing the differences, 
but it still shows the important parts:
One or two columns
Vertical Caption bar
Lines= text
Circle = picture
Squares= icons

Within a couple of seconds, I can see the layouts match up.
